site/docs/skylark/backward-compatibility.md

```diff
@@ -1,15 +1,12 @@
 ---
 layout: documentation
-title: Backward Compatibility
+title: Starlark Incompatible CHnages
```

Comment:
This page also contains rule specific changes.

We should decide what we want to do with this page: either mention all incompatible changes, or kill it. The current state is misleading.

Reply:
What would you like to do?